backend/internal/github/appclient/appclient.go

```diff
@@ -56,7 +60,7 @@ func (api *AppAPI) GetJWT(ctx context.Context) (string, error) {
 	return token.AccessToken, nil
 }
 
-func (api *AppAPI) ListInstallations(ctx context.Context) ([]*github.Installation, error) {
+func (api *AppAPI) GetClientWithJWTAuth(ctx context.Context) (*github.Client, error) {
```

Comment:
You shouldn't be returning a github client, this should be a struct field.

Think of it as a private field in Java with a wrapper class.

Okay as a private method if you rename with lowercase G